api/src/main/java/com/messagebird/objects/MessageReference.java

```diff
@@ -4,6 +4,7 @@
 
     private String href;
     private int totalCount;
+    private String lastMessageId;
```

Comment:
I realize that this object is also used for `Contact`. I think it might be irrelevant to have in `Contact` object. What do you think?

Reply:
I see. One option is to set it as a `Optional` field and reuse for both, the other is to split classes. I think both are clean solutions.

Reply:
Or I will keep it as it is, as object mapper will keep it as null if not present. Making the field optional doesn't sound right, on a second thought, as it might be: `null`, `Optional.empty()`, `Optional.of("some_id")`. Further complicating the issue.

Reply:
Kept as is.